src/agent/rustjail/src/container.rs

```diff
@@ -1540,14 +1545,26 @@ async fn execute_hook(logger: &Logger, h: &Hook, st: &OCIState) -> Result<()> {
             .unwrap();
         info!(logger, "child stdout: {}", out.as_str());
 
+        let mut err = String::new();
```

Comment:
@Bevisy you added reading stderr code here, so can you also add a test case that runs invalid hooks and generate stderr?

Reply:
Thank you for your suggestion, I will deal with it later

Reply:
I have add the test which can redirect logs to stderr, but this test looks awkward. Because it looks like doing nothing. Can you give any advice?